server/images.go

```diff
@@ -1162,3 +1164,24 @@ func verifyBlob(digest string) error {
 
 	return nil
 }
+
+var isContainer = ""
```

Comment:
If this is being added directly to the user agent, it might be better to call this something like `deployment` or `variant`